src/lazy/encoder/writer.rs

```diff
@@ -1435,5 +1441,33 @@ mod tests {
             assert!(eexp_writer.write("hello").is_err());
             Ok(())
         }
+
+        #[test]
+        fn tagless_uint8_encoding() -> IonResult<()> {
```

Comment:
Is this also implemented for tagless uint8 expression groups? If so, can you add a test case covering that?